ProjectLighthouse/Pages/SlotPage.cshtml

```diff
@@ -53,14 +57,84 @@
             }
         </div>
     </div>
-</div>
+    <div class="eight wide column">
+        @await Html.PartialAsync("Partials/CommentsPartial")
+    </div>
+    <div class="eight wide column">
+        <div class="ui purple segment">
+            <h2>Reviews</h2>
+            @if (Model.Reviews.Count == 0 && Model.ReviewsEnabled)
+            {
+                <p>There are no reviews.</p>
+            }
+            else if (!Model.ReviewsEnabled)
+            {
+                <b>
+                    <i>Reviews are disabled on this level.</i>
+                </b>
+            }
+            else
+            {
+                int count = Model.Reviews.Count;
+                <p>There @(count == 1 ? "is" : "are") @count review@(count == 1 ? "" : "s").</p>
+            }
+            <div class="ui divider"></div>
 
-@await Html.PartialAsync("Partials/CommentsPartial")
+            @foreach (Review review in Model.Reviews)
+            {
+                string faceHash = review.Thumb switch {
+                    -1 => review.Reviewer.BooHash,
+                    0 => review.Reviewer.MehHash,
+                    1 => review.Reviewer.YayHash,
+                    
+                    _ => throw new ArgumentOutOfRangeException(),
```

Comment:
The /postReview endpoint doesn't clamp the Thumb value so a user can make their profile cause an error. This should either have a default value or the Thumb value needs to properly get limited in ReviewController.

Reply:
Good to know. Are there any other places you can think of where ratings aren't clamped?

Reply:
In my purge fixes branch I already fixed comment ratings not being clamped, but you could fix it here if you wanted

Reply:
Hmm. If that's on a separate commit I could cherry-pick it into master. If not I can fix it in this MR. Up to you.

Reply:
It'll be one big commit so I'll just do it in my PR.

Reply:
Cool. I'll wait until then.